Automattic/wp-calypso
# ActivityLog: prevent 'Copy site' button from wrapping text

Fixes https://github.com/Automattic/jetpack-backup-team/issues/178

## Proposed Changes

* Replace `Copy this site` with `Copy site`, removing the redundant `this`
* Prevent 'Copy site' button from wrapping text
* Responsive adjustments for filter bar on mobile devices on Jetpack Cloud

| Page | Before | After |
|---|---|---|
| /activity-log/ | ![image](https://github.com/Automattic/wp-calypso/assets/1488641/49875ba8-1bd7-4cde-accc-96f92abea691) | ![CleanShot 2023-10-31 at 12 39 13@2x](https://github.com/Automattic/wp-calypso/assets/1488641/fc54eac9-b174-4c13-9d8c-bae4b9b1f52e) |
| /backup/ | ![CleanShot 2023-10-30 at 21 33 01@2x](https://github.com/Automattic/wp-calypso/assets/1488641/a4e4b1d6-f84f-4e4e-bf82-3ba49116cbdb) | ![CleanShot 2023-10-31 at 12 39 29@2x](https://github.com/Automattic/wp-calypso/assets/1488641/450dd337-f716-4404-a115-b85c86ecd824) |

## Testing Instructions
* Spin up a Jetpack Cloud Live Branch (this button is not available in Calypso Blue)
* Visit the Activity Log and Backup page and ensure you see the `Copy site` button on the top right
* Using your developer tools, ensure that `Copy site` doesn't break into a new line while testing with mobile dimensions

## Pre-merge Checklist
